src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -776,6 +831,78 @@ static int Lambda(int x, Func<int, int> l)
 True
 True
 True
+True
+";
+
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe);
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.DebugExe);
+        }
+
+        [Fact]
+        public void MultipleFilesCoverage()
```

Comment:
Personally, I'm not worried about supporting more than 2 billion documents. If that's a concern, this isn't the only place that will need to start using 64-bit indices/IDs.